modules/custom/openy_campaign/src/Controller/CampaignReportsController.php

```diff
@@ -9,6 +9,9 @@
 use Drupal\Core\Url;
 use Drupal\node\Entity\Node;
 use Drupal\node\NodeInterface;
+use Drupal\openy_campaign\OpenYLocaleDate;
+use Drupal\openy_popups\Form\ClassBranchesForm;
+use Drupal\openy_session_instance\SessionInstanceManager;
```

Comment:
Please add respective dependencies to the module info file https://github.com/kolesnikoff/openy/blob/YM-63-live-scorecard/modules/custom/openy_campaign/openy_campaign.info.yml